packages/wallets/src/components/offers/OfferImport.tsx

```diff
@@ -7,6 +7,7 @@ import { useGetOfferSummaryMutation } from '@chia/api-react';
 import { type OfferSummaryRecord } from '@chia/api';
 import OfferDataEntryDialog from './OfferDataEntryDialog';
 import fs, { Stats } from 'fs';
+import { useHotkeys } from 'react-hotkeys-hook';
```

Comment:
Is this dependency already present in main? If not, I think running `npx lerna add react-hotkeys-hook --scope @chia/gui` and then adding the package.json/package-lock.json files should do it.

Reply:
good catch, i forgot to include those files in the commit